### PR TITLE
OEL-2794: Update components to run on Drupal 10.2.x

### PR DESCRIPTION
**Pull request by AaronGilMartinez**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "require": {
         "php": ">=8.1",
         "cweagans/composer-patches": "^1.7",
-        "drupal/core": "^9.5 || ^10",
+        "drupal/core": "^10.1 || ^10.2",
```
Comment (brummbar) [inline composer.json]: Let's just put ^10 here, without further restrictions.

File: composer.json [hunk 2]
```diff
@@ -18,10 +18,10 @@
     "require-dev": {
         "composer/installers": "^2.0",
         "drupal/config_devel": "^1.9",
-        "drupal/core-composer-scaffold": "^9.4 || ^10",
-        "drupal/core-dev": "^9.4 || ^10",
+        "drupal/core-composer-scaffold": "^10.1 || ^10.2",
```
Comment (brummbar) [inline composer.json]: Same here.

**Review by brummbar — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (AaronGilMartinez) [reply composer.json]: Done

File: composer.json (hunk 2 above)
Comment (AaronGilMartinez) [reply composer.json]: Done

**Review by brummbar — APPROVED** [review]

**Pull request merged**